### ISSUE TITLE
No network request being made on widget update

### ISSUE THREAD

**Issue by joaosoares** [algolia/react-instantsearch#53]
**Do you want to request a *feature* or report a *bug*?**
Bug

**Bug: What is the current behavior?**
Whenever I click on a refinement, the hits do not update. If I go to the network tab, I see no request is made to the server.

**Bug: What is the expected behavior?**
Whenever I click on a refinement, a network request to Algolia is made and the hits are updated.

**What is the version you are using? Always use the latest one before opening a bug issue.**
4.0.0-beta3

Here's a sample code that illustrates how I'm using the Widgets:
```jsx
<InstantSearch
  appId={/*...*/}
  apiKey={/*...*/}
  indexName={/*...*/}
>
  <SearchBox />
  <CurrentRefinements />
  <Menu attributeName="option_values.name" />
  <ClearAll />
  <Stats />
  <Hits />
  <Pagination showLast={true} />
</InstantSearch>
```

**Comment by joaosoares** [comment]
Investigating further, the issue seems to be with `attributeName` accessing a subproperty. If I change it to, say, `price`, it works as expected. Am I missing something wrt attribute naming?

**Comment by mthuret** [comment]
Hi @joaosoares, can you try with `<Menu attributeName={option_values.name} />`?

If it's still not working, please use this [jsfiddle](https://jsfiddle.net/vvoyer/ar80rgpc/2/) to reproduce your issue.

**Comment by joaosoares** [comment]
Hi @mthuret! Sorry for the delay. Writing it that way gives me a JavaScript error, as there's no variable named option_values (that's just the refinement name).

I have tried to reproduced the issue in the jsfiddle you provided, but could not reproduce it. Then I created a sample app with create-react-app and also could not reproduce it. This leads me to believe that it is a weird issue on our application's end. I think the Issue can be closed, and if I ever find a solution besides not using compound attribute names, I will inform it here. Thank you for your attention.
